.chloggen/fix_red-metrics.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
```

Comment:
Is the fix also fixing monolithic?

Reply:
For me it looks like that was never supported? I can not find anything about the `STORAGE_METRICS_TYPE`.

Reply:
cc) @andreasgerstmayr

Reply:
yes, monolithic doesn't support red metrics at the moment